mono/mini/mini-exceptions.c

```diff
@@ -133,6 +133,7 @@ static void mono_summarize_managed_stack (MonoThreadSummary *out);
 static void mono_summarize_unmanaged_stack (MonoThreadSummary *out);
 static void mono_summarize_exception (MonoException *exc, MonoThreadSummary *out);
 static void mono_crash_reporting_register_native_library (const char *module_path, const char *module_name);
+static void mono_crash_reporting_allow_all_native_libraries (void);
```

Comment:
Windows builds are unhappy about this declaration (should be under the same ifdef as the definition):

```
..\mono\mini\mini-exceptions.c(4004): error C2129: static function 'void mono_crash_reporting_allow_all_native_libraries(void)' declared but not defined
```